python-ogr.spec

```diff
@@ -4,7 +4,7 @@
 %{?python_enable_dependency_generator}
 
 Name:           python-%{pypi_name}
-Version:        0.5.0
+Version:        0.6.0
```

Comment:
Is version change needed?

Reply:
yes, AFAIK

Reply:
the build would fail if you didn't do that